modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/TypeScriptAngularClientCodegen.java

```diff
@@ -233,7 +233,7 @@ public void processOpts() {
             additionalProperties.put(HTTP_CONTEXT_IN_OPTIONS, true);
         }
 
-        if (ngVersion.atLeast("17.0.0")) {
```

Comment:
please keep that, since it is needed for ng 17 as well

Reply:
Oops, my mistake. I reverted the wrong adjustment and re-ran the build scripts. Thank you for your feedback.